docs/source/modules.rst

```diff
@@ -0,0 +1,146 @@
+Modules
+=======
+
+While different modules perform different jobs, their interfaces all follow
+the same pattern as much as possible.
+
+The API of each module is composed of two parts. The *auth* parameter contains
+the pieces of information that are related to the Sensu Go backend that the
+module is connecting to. All other parameters hold the information related to
+the resource that we are operating on. Typical task would look like this:
+
+.. code-block:: yaml
+
+   - name: Make sure asset is present
+     asset:
+       auth:
+         url: http://sensu.host.com:8080
+         user: my-user
+         password: pass
+       namespace: my-namespace
+       name: my-asset-name
+       builds: ...
+
+Almost every module for manipulating resources has its info counterpart that
```

Comment:
I would rephrase this a little, something along the lines: "Almost every module for manipulating resources has its counterpart module that can be used to retrieve information about the corresponding resources, for instance _asset_ and _asset_info_ modules".

I think that "has its info counterpart" is a bit confusing for someone not familiar with the collection.